src/branch_tests.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html>
+<head>
+  <title>Tests for Branch</title>
+</head>
+<body>
+  <script src="../../closure-library/closure/goog/base.js"></script>
+  <script src="../../closure-library/closure/goog/testing/testcase.js"></script>
+  <script src="../../closure-library/closure/goog/testing/testrunner.js"></script>
+  <script src="../../closure-library/closure/goog/testing/propertyreplacer.js"></script>
+  <script src="../../closure-library/closure/goog/testing/continuationtestcase.js"></script>
+  <script src="branch-deps.js"></script>
+  <script src="2_branch_tests.js"></script>
+  <script>
+  	goog.require('Branch');
+    goog.require('utils');
```

Comment:
You know what, I don't need utils - not sure why I put it there :-P
and ya, 2_branch_tests is magic!